.github/markdown-link-check.jsonc

```diff
@@ -7,4 +7,8 @@
       }
     }
   ],
+  "ignorePatterns": [
+    { "pattern": "^#" },
+    { "pattern": "https://www\\.st\\.com/en/development-tools/hardware-debugger-and-programmer-tools-for-stm32/products\\.html" }
```

Comment:
Is this a broken link and does it need fixing somewhere?

Reply:
The request to `https://www.st.com/... `failed due to the server forcefully closing the connection (a "socket hang up"), which results in an ECONNRESET error from Node.js
This usually happens when:
- **Rate limiting or bot protection**, STMicroelectronics' website might be blocking automated requests or rate-limiting connections from CI systems or headless HTTP clients.
- **Temporary server issues**: This is not the case, As I manually checked the URL a couple of times.

Reply:
OK, thanks for the explanation. Makes sense.